### DISCUSSION TITLE
Issues loading Addressables from remote location

### DISCUSSION THREAD

**Discussion by junniepat** [jeffreylanters/react-unity-webgl#217] [Help Wanted!] [answered]
While trying to load in addressables i get this error.
I also noticed my streaming assets are not going to the root folder.

Below is a picture of the error i am getting.

![image](https://user-images.githubusercontent.com/3358365/127870264-a726a67c-2b29-4270-a580-e3327430ae17.png)

**Comment by jeffreylanters** [comment, marked as answer]
Can you share a snippet of your implementation?

**Reply by junniepat** [reply]
Ok, so I figured out that I needed to define the StreamAssetsUrl in the my file where I'm referencing the modules

const unityContext = new UnityContext({
    loaderUrl: "/WebGl/Stands/Build/Build.loader.js",
    dataUrl: "/WebGl/Stands/Build/Build.data",
    frameworkUrl: "/WebGl/Stands/Build/Build.framework.js",
    codeUrl: "/WebGl/Stands/Build/Build.wasm",
    streamingAssetsUrl: '/WebGl/Stands/StreamingAssets'
});

**Reply by Dreitser** [reply]
Hey @junniepat @jeffreylanters can you share with us best practices on how to implement please?

**Comment by aidie8** [comment]
Thank you so much for this, i have spent so many hours looking for this fix,
